api/all/src/main/java/io/opentelemetry/api/common/AttributesBuilder.java

```diff
@@ -36,22 +35,18 @@ public interface AttributesBuilder {
   // version.
   <T> AttributesBuilder put(AttributeKey<Long> key, int value);
 
-  /**
-   * Puts an {@link AttributeKey} with an associated value into this if the value is non-null.
-   * Providing a null value does not remove or unset previously set values.
```

Comment:
do you see an issue with keeping the Javadoc changes (which reflect the existing behavior for Java at least)